src/Infrastructure/ifcviewerExtensions/IfcViewerAPIExtended.js

```diff
@@ -0,0 +1,56 @@
+import {IfcViewerAPI} from 'web-ifc-viewer'
+
+/**
+ * Extending the original IFCViewerFunctionality
+ */
+export class IfcViewerAPIExtended extends IfcViewerAPI {
+  // TODO: might be usefull if we used a Set as well to handle large selections,
+  // but for now array is more performant for small numbers
+  #selectedExpressIds = []
```

Comment:
What is the hashtag syntax for?

Reply:
Declaring the array as private member to prevent directly changing it from outside class